lib/omnibus/packagers/rpm.rb

```diff
@@ -641,7 +647,11 @@ def install(packages, enablerepo = NULL)
     # @return [void]
     #
     def remove(packages)
-      `yum -y remove #{packages}`
+      if ohai["platform_family"] == 'suse'
+        `zypper remove -y #{packages}`
```

Comment:
while you're at it, could you use `shellout!` instead here? (and 2 lines below too)